pkg/apis/internal.admin.acorn.io/v1/quotarequests_test.go

```diff
@@ -337,6 +337,41 @@ func TestEquals(t *testing.T) {
 			},
 			expected: true,
 		},
+		{
+			name: "unequal QuotaRequestResources only",
+			current: QuotaRequestResources{
+				BaseResources: BaseResources{
+					Apps:          1,
+					VolumeStorage: resource.MustParse("1Mi"),
+				},
+				Secrets: 1,
+			},
+			incoming: QuotaRequestResources{
+				BaseResources: BaseResources{
+					Apps:          1,
+					VolumeStorage: resource.MustParse("1Mi"),
+				},
+			},
+			expected: false,
+		},
+		{
+			name: "unequal base resources only",
+			current: QuotaRequestResources{
+				BaseResources: BaseResources{
+					Apps:          1,
+					Containers:    1,
+					VolumeStorage: resource.MustParse("1Mi"),
+				},
+				Secrets: 1,
```

Comment:
remove in order for this test to do what we want it to

```suggestion
```

Reply:
You're right that something needs to change here, but I actually want to add the `Secrets: 1` to the incoming.